### DISCUSSION TITLE
Providing z/OS Tools and Toys in zopen

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#682] [General]
As @AnthonyGiorgio pointed out, there are number open source tools under https://github.com/IBM/IBM-Z-zOS/tree/main/zOS-Tools-and-Toys .

Should we bundle these into one package or provide them as individual packages?

Some packages collide with GNU tools like which, where, and stat. Which tools in z/OS Tools and Toys should be expose first? Are there any particular ones that are really useful?

**Comment by lbdyck** [comment]
The goal is to provide the community with access to tools (and toys) that will make them productive using z/OS - thus I'm in favor of any/all packages.

**Comment by AnthonyGiorgio** [comment]
How should we manage the tools from a zopen perspective?  If they are all in one bucket, what should happen when I update a tool, or add a new one?  I just added `users.rexx`, and I have an update planned for `sparse`.  Would a change to one tool trigger a new package version?

**Reply by IgorTodorovskiIBM** [reply]
It would trigger a new release. The package version update would be up to the maintainer of the bundle.

How often are they updated? 

If we bundle them, it would be similar to coreutils or moreutils. Those projects though are much more stable.

We could also bundle them based on category (file I/O, system tools, networking, etc).
